pkg/conf/natsconf.go

```diff
@@ -0,0 +1,73 @@
+// natsconf is a package for producing NATS config programmatically.
+package natsconf
```

Comment:
Long term, we'll want to find a way to more closely integrate with NATS server, but certainly OK for now.

Reply:
Agreed, should find a better way here eventually